tests/ssg_test_suite/rule.py

```diff
@@ -412,9 +412,10 @@ def _check_and_record_rule_scenario(self, scenario, remote_rule_dir, rule_id, re
 
     @contextlib.contextmanager
     def copy_of_datastream(self, new_filename=None):
+        prefixed_name = common.get_prefixed_name("ds_modified")
         old_filename = self.datastream
         if not new_filename:
-            descriptor, new_filename = tempfile.mkstemp(prefix="ssgts_ds_modified", dir="/tmp")
+            descriptor, new_filename = tempfile.mkstemp(prefix=prefixed_name, dir="/tmp")
```

Comment:
this is generating "__ssgts_ds_modified" based on get_prefixed_name function. I don't if that will be a problem.

Reply:
I checked this change with QEs before apply it. We didn't find any problem.